### ISSUE TITLE
Switch between application tabs

### ISSUE THREAD

**Issue by ikeedo** [rbreaves/kinto#173]
**Is your feature request related to a problem? Please describe.**
I am used to tab switching with cmd+alt+left/right in MacOS applications like Brave Browser and VSCode.

**Describe the solution you'd like**
Would it be possible for kinto to integrate these left/right tab switching shortcuts for such applications?

**Describe alternatives you've considered**
Alternatively, I could try to change those keyboard shortcuts from within the apps. But that my be missing the point, since a global scoped key mapping would be better in my opinion.

**Comment by rbreaves** [comment]
Yea, should be possible I think it is just a matter of remapping those keys to Ctrl-PgUp/PgDwn.

**Comment by rbreaves** [comment]
The new keymap has been added to the dev branch. Not sure when I will merge it to master, but it will be merged whenever I decide to update master next. I actually quite enjoy this new keymap.. even more so than the actual Ctrl-Tab for tab switching.. tempted to swap it out for this one, but that would then deviate from default behaviors which I am not going to do.

What I might do personally keymap wise will not usually make it into this repo. Although I have a few sublime text specific keymaps that I have enabled to work under VS Code (assuming it does not conflict with an existing VS Code hotkey - a feature I don't often mention lol). I might go in later and disable those by default like I do under the Windows implementation, I ask the user if they want Sublime text like keymaps as well for VS Code - but it is not a complete remap just yet, just the keys that matter to me the most.

Closing this ticket for now, if dev doesn't do it for you just let me know.
